language/stdlib/src/lib.rs

```diff
@@ -59,7 +59,65 @@ pub fn filter_move_files(dir_iter: impl Iterator<Item = PathBuf>) -> impl Iterat
     })
 }
 
-pub fn stdlib_files() -> Vec<String> {
+pub fn filter_move_bytecode_files(
+    dir_iter: impl Iterator<Item = PathBuf>,
+) -> impl Iterator<Item = PathBuf> {
+    dir_iter.flat_map(|path| {
+        if path.extension()?.to_str()? == COMPILED_EXTENSION {
+            Some(path)
+        } else {
+            None
+        }
+    })
+}
+
+fn stdlib_modules() -> Vec<&'static str> {
```

Comment:
could we break this out into a separate PR?

Reply:
Good call; I'll just use the whole Libra Framework as the default `move_lib` for now (which might make sense anyway)